pkgs/top-level/python-packages.nix

```diff
@@ -3846,36 +3850,17 @@ in {
     };
   };
 
+  colorama = callPackage ../development/python-modules/colorama { };
 
-  colorama = buildPythonPackage rec {
-    name = "colorama-${version}";
+  colorama37 = self.colorama.overrideAttrs (old: rec {
```

Comment:
Ok. I could probably remove the check since the release it's just bugfixes but the developer of awscli says colorama had been breaking the program before.